int32 battery_percentage
float32 Ax
float32 Ay
float32 Az
float32 battery_temp
float32 voltage
